lib/foreman_maintain/utils/backup.rb

```diff
@@ -19,26 +19,16 @@ def initialize(backup_dir)
         @foreman_offline_files = ['pgsql_data.tar.gz']
         @katello_online_files = ['candlepin.dump', 'foreman.dump']
         @katello_offline_files = ['pgsql_data.tar.gz']
-        if feature(:pulp2)
-          @katello_online_files  << 'mongo_dump'
-          @katello_offline_files << 'mongo_data.tar.gz'
-          @fpc_online_files = ['mongo_dump']
-          @fpc_offline_files = ['mongo_data.tar.gz']
-        elsif feature(:pulpcore_database)
-          @katello_online_files << 'foreman.dump'
```

Comment:
while this line was wrong, I think `katello_online_files` should include `pulpcore.dump` ;)